sdk/storage/inc/shares/share_constants.hpp

```diff
@@ -0,0 +1,19 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+namespace Azure { namespace Storage { namespace Files { namespace Shares {
+
+  constexpr static const char* c_FileDefaultTimeValue = "now";
+  constexpr static const char* c_FileDefaultPermission = "inherit";
```

Comment:
I don't think `default` is a good name. For copy operation, preserve is default, while for create operation, inherit is default.

Reply:
Changed to `c_FileInheritPermission`.